x-pack/plugins/security_solution/common/endpoint/service/host_isolation/utils.ts

```diff
@@ -12,19 +13,14 @@ export const isVersionSupported = ({
   currentVersion: string;
   minVersionRequired: string;
 }) => {
-  const parsedCurrentVersion = currentVersion.includes('-SNAPSHOT')
+  const parsedCurrentVersion = currentVersion.includes('-')
```

Comment:
Q. Do you even have to parse it?

and if so, maybe use `semverParse` from `semver` lib? (note: I did not look at it and what it does, but its there). They also have validators for the strings to ensure they are valid semantic versions

Reply:
Some parsing is required in order to ensure that the passed in version is equal to the `minVersion`.  With the expanded tests, I feel better about this approach.